src/main/java/nl/knaw/dans/ingest/core/AbstractIngestArea.java

```diff
@@ -28,12 +28,19 @@
 
 public class AbstractIngestArea {
     private static final Logger log = LoggerFactory.getLogger(AbstractIngestArea.class);
+
+
+
     protected final Path inboxDir;
     protected final Path outboxDir;
     protected final DepositIngestTaskFactory taskFactory;
     protected final TaskEventService taskEventService;
     protected final EnqueuingService enqueuingService;
 
+    protected Path getInboxDir() {
```

Comment:
Why was this method added?  `inboxDir` is `protected` so can be reached from sub-classes anyway.